apis/sgv2-docsapi/src/test/java/io/stargate/sgv2/docsapi/api/v2/namespaces/collections/CollectionsResourceIntegrationTest.java

```diff
@@ -166,7 +165,7 @@ public void malformedPayload() {
           .then()
           .statusCode(400)
           .body("code", is(400))
-          .body("description", startsWith("Unable to process JSON"));
+          .body("description", is("HTTP 400 Bad Request"));
```

Comment:
Wouldn't we ideally want something more specific to indicate bad JSON?

Reply:
This was coming from the `2.15.1` update, not this issue.. We can not do anything, as Quarkus is short-circuiting this now and throwing `WebApplicationException` instead of the `JsonParseException` as before..

Reply:
> This was coming from the `2.15.1` update, not this issue.. We can not do anything, as Quarkus is short-circuiting this now and throwing `WebApplicationException` instead of the `JsonParseException` as before..

Ok.